src/doc/unstable-book/src/compiler-flags/wasm-c-abi.md

```diff
@@ -3,8 +3,8 @@
 This option controls whether Rust uses the spec-compliant C ABI when compiling
 for the `wasm32-unknown-unknown` target.
 
-This makes it possible to be ABI-compatible with all other spec-compliant Wasm
-like Rusts `wasm32-wasip1`.
+This makes it possible to be ABI-compatible with all other spec-compliant WASM targets
```

Comment:
- `WASM` is an abbreviation for WebAssembly.
- The addition of `targets` clarifies what is described after in the sentence.
- `Rust's` vs `Rusts` signifies ownership of the later mentioned `wasm32-wasip1` target name.

Reply:
It's "Wasm", not "WASM": https://webassembly.github.io/spec/core/intro/introduction.html#wasm. The other two changes seem fine

Reply:
Nice, thanks for the link. Revert it back to "Wasm" :)